webapp/src/codecard.tsx

```diff
@@ -118,6 +118,9 @@ export class CodeCardView extends data.Component<pxt.CodeCard, CodeCardState> {
             {card.time ? <div className="meta">
                 {card.tutorialLength ? <span className={`ui tutorial-progress ${tutorialDone ? "green" : "orange"} left floated label`}><i className={`${tutorialDone ? "trophy" : "circle"} icon`}></i>&nbsp;{lf("{0}/{1}", (card.tutorialStep || 0) + 1, card.tutorialLength)}</span> : undefined}
                 {card.time ? <span key="date" className="date">{pxt.Util.timeSince(card.time)}</span> : null}
+                {card.cloudState === "cloud" &&
+                    <i className="ui large left floated icon cloud"></i>
```

Comment:
I think this is where we could hook into the conflict/sync state of the project, via the virtual api. Add a TODO for it?

Reply:
Exactly. Will do!